dist/index.d.ts

```diff
@@ -9,23 +9,24 @@
 interface HTMLCanvasElement {
   getContext(
     contextId: "gpupresent"
-  ): GPUCanvasContext | null;
+  ):
+    | GPUPresentationContext
+    | GPUCanvasContext
```

Comment:
Thanks! Wasn't sure how this worked in TS.

Reply:
Tested this locally in a CTS build and it turns out the intersection ends up being empty because of the `__brand` thing. Pushed a commit that fixes that

Reply:
Oops this already landed, I'll open another PR